ncard_app/urls.py

```diff
@@ -8,17 +8,44 @@
     path('dashboard/', views.dashboard, name="dashboard"),
     path('login_user/', auth_views.LoginView.as_view(), name="login"),
     path('logout_user/', auth_views.logout_then_login, name="logout"),
-    path("test/show/",views.test_show,name='fuzzy'),
 
-    path('tables/people/', views.list_people, name="list-people"),
+    path('tables/people/', login_required(views.FilteredPersonListView.as_view()), name="list-people"),
     path('tables/people/add/', login_required(views.PersonCreateView.as_view()), name="add-person"),
     path('tables/people/<int:pk>/', login_required(views.PersonUpdateView.as_view()), name="edit-person"),
 
-    path('tables/organisations/', views.list_organisations, name="list-organisations"),
+    path('tables/organisations/', login_required(views.FilteredOrganisationListView.as_view()), name="list-organisations"),
     path('tables/organisations/add/', login_required(views.OrganisationCreateView.as_view()), name="add-organisation"),
     path('tables/organisations/<int:pk>/', login_required(views.OrganisationUpdateView.as_view()), name="edit-organisation"),
 
+    path('tables/awards/',login_required(views.FilteredAwardListView.as_view()), name="list-awards"),
+    path('tables/awards/add/', login_required(views.AwardCreateView.as_view()), name="add-award"),
+    path('tables/awards/<int:pk>/', login_required(views.AwardUpdateView.as_view()), name="edit-award"),
+
+    path('tables/events/', login_required(views.FilteredEventsListView.as_view()), name="list-events"),
+    path('tables/events/add/', login_required(views.EventCreateView.as_view()), name="add-events"),
+    path('tables/events/<int:pk>/', login_required(views.EventUpdateView.as_view()), name="edit-events"),
```

Comment:
Should be add-event and edit-event, singular

Reply:
changed it